### PR TITLE
[RW-5388][risk=no] Puppeteer enable failed test for Export to Notebook

### PR DESCRIPTION
**Pull request by aweng98**
Export to Notebook thru ellipsis menu is working again. Enable automated test.
https://precisionmedicineinitiative.atlassian.net/browse/RW-5388

### CODE REVIEW

File: e2e/tests/datasets/export-py-notebook.spec.ts [hunk 1]
```diff
@@ -110,14 +103,22 @@ describe('Create Dataset', () => {
...
-    const cardExists = await resourceCard.cardExists(notebookName, ResourceCard.Notebook);
-    expect(cardExists).toBe(true);
+    // Verify notebook created successfully. Not going to start the Jupyter notebook.
+    const notebookPreviewPage = new NotebookPreviewPage(page);
+    await notebookPreviewPage.waitForLoad();
+    const currentPageUrl = page.url();
+    expect(currentPageUrl).toContain(`notebooks/preview/${notebookName}.ipynb`);
+
+    // Navigate to Workpace Notebooks page.
+    const analysiPage = await notebookPreviewPage.goAnalysisPage();
```
Comment (jmthibault79) [inline e2e/tests/datasets/export-py-notebook.spec.ts:113]: typo

File: e2e/tests/datasets/export-py-notebook.spec.ts [hunk 2]
```diff
@@ -54,11 +52,7 @@ describe('Create Dataset', () => {
     expect(code).toContain('import os');
 
     // Navigate to Workpace Notebooks page.
-    const notebooksLink = await Link.findByName(page, {name: 'Notebooks'});
-    await notebooksLink.clickAndWait();
-
-    const analysiPage = new WorkspaceAnalysisPage(page);
-    await analysiPage.waitForLoad();
+    const analysiPage = await notebookPreviewPage.goAnalysisPage();
```
Comment (jmthibault79) [inline e2e/tests/datasets/export-py-notebook.spec.ts:55]: typo

**Review by jmthibault79 — APPROVED** [review]
looks good, except for the typos

File: e2e/tests/datasets/export-py-notebook.spec.ts (hunk 2 above)
Comment (aweng98) [reply e2e/tests/datasets/export-py-notebook.spec.ts]: Fixed typo.

**Pull request merged**